wikidev/wikidev/views.py

```diff
@@ -28,68 +34,82 @@ def get_user(self):
         data = jwt.decode(token.replace('Bearer ', ''), key='87654321')
         user, created = User.objects.get_or_create(id=data['sub'],
                                                    username=str(data['sub']))
-       # user, created = User.objects.get_or_create(id=231243,
-       #                                            username='Mircea')
         return user
 
-    def to_response(self, menu_or_form):
-        response = onem.Response(
-                menu_or_form, self.request.headers['X-Onem-Correlation-Id']
-        )
-        return HttpResponse(response.as_json(), content_type='application/json')
+    def to_response(self, content):
+        response = Response(content=content)
+        response.correlation_id = self.request.headers['X-Onem-Correlation-Id']
+
+        return HttpResponse(response.json(), content_type='application/json')
 
 
 class HomeView(View):
     http_method_names = ['get']
 
     def get(self, request):
-        #user = self.get_user()
-
-        body = [
-            onem.menus.MenuItem('Search', url=reverse('search_wizard')),
-            onem.menus.MenuItem('Random', url=reverse('random')),
-            onem.menus.MenuItem('Language', url=reverse('language'))
+        #user = self.get_user()  # TODO
+
+        menu_items = [
+            MenuItem(type=MenuItemType.option,
```

Comment:
Just an idea to improve the sdk usage by looking at the code: removing `type` param from constructor because the `type`  can be easily determined like so:
1. If a `path` is given, then `type` must be `option`
2. If no `path` is given, then `type` must be content.
What do you think?

Reply:
Yeah, makes sense, if we have a path(an URL) then the type is a selectable option, otherwise it is just plain content.